packages/lesswrong/components/common/TabNavigationMenu/menuTabs.tsx

```diff
@@ -330,7 +330,7 @@ export const menuTabs: ForumOptions<Array<MenuTab>> = {
       showOnCompressed: true
     }, {
       id: 'community',
-      title: 'Join a group',
+      title: 'Groups directory',
```

Comment:
Throwing this tiny change in - the copy actually made the click rate worse, so trying a new one